view/theme/frio/templates/profile/schedule.tpl

```diff
@@ -0,0 +1,24 @@
+<div class="generic-page-wrapper">
+	{{* include the title template for the settings title *}}
+	{{include file="section_title.tpl" title=$title}}
+	<form action="profile/{{$nickname}}/schedule" method="post" autocomplete="off">
+		<input type='hidden' name='form_security_token' value='{{$form_security_token}}'>
+		<table id='application-block' class='table table-condensed table-striped'>
+			<thead>
+				<tr>
+					<th>{{$scheduled_at}}</th>
+					<th>{{$content}}</th>
+				</tr>
+			</thead>
+			<tbody>
+				{{foreach $schedule as $row}}
+				<tr>
+					<td>{{$row.scheduled_at}}</td>
+					<td>{{$row.content}}</td>
+					<td><a href="{{$baseurl}}/profile/{{$nickname}}/schedule/delete/{{$row.id}}?t={{$form_security_token}}" class="btn" title="{{$delete}}"><i class="fa fa-trash" aria-hidden="true"></i></a></td>
```

Comment:
Please replace this link with a form with a security token and a submit button.

Reply:
Du you have an example for this? At several places (oauth, users) we seem to make it via some `get` action.

Reply:
In this case:
```suggestion
					<td>
						<form action="{{$baseurl}}/profile/{{$nickname}}/schedule" method="post">
							<input type="hidden" name="form_security_token" value="{{$form_security_token}}">
							<button type="submit" name="delete" value="{{$row.id}}" title="{{$delete}}">
								<i class="fa fa-trash" aria-hidden="true">
							</button>
						</form>
					</td>
```

Reply:
A form per single line?